src/bootstrap-window.js

```diff
@@ -239,7 +235,7 @@
 		if (safeProcess.platform === 'win32' && pathName.startsWith('//')) { // specially handle Windows UNC paths
 			uri = encodeURI(`file:${pathName}`);
```

Comment:
What about this use of `file` protocol?

Reply:
I have to check how UNC path works with custom protocol, didn't want to update this before that.